python/cudf/cudf/core/frame.py

```diff
@@ -1701,9 +1702,10 @@ def _colwise_binop(
                     # that nulls that are present in both left_column and
                     # right_column are not filled.
                     if left_column.nullable and right_column.nullable:
-                        lmask = as_column(left_column.nullmask)
-                        rmask = as_column(right_column.nullmask)
-                        output_mask = (lmask | rmask).data
+                        with acquire_spill_lock():
+                            lmask = as_column(left_column.nullmask)
+                            rmask = as_column(right_column.nullmask)
+                            output_mask = (lmask | rmask).data
```

Comment:
Did we miss this in the get_ptr refactor?

Reply:
Yup